tests/functional_tests/iterator_tests/test_trivial.py

```diff
@@ -167,3 +167,30 @@ def test_vertical_shift_unstructured(program_processor):
 
     if validate:
         assert np.allclose(inp_s[:, 1:], np.asarray(out_s)[:, :-1])
+
+
+# TODO We should bypass all passes to ensure that we actually test this code as is
```

Comment:
Currently, this test does not test an non-applied operator because we aggressively inline and we don't have easy access to disable passes.